src/app/js/lib/getTitle.js

```diff
@@ -1,13 +1,26 @@
 import { getCleanHost } from '../../../common/uris';
 
+const HOST_REGEX = /https?:\/\/([\w-]+)/;
+
 export default () => {
     const host = getCleanHost();
+
     /**
-     * @type {string}
+     * @type {string[]}
      */
-    const tenant = window.__TENANT__ ?? 'default';
+    const title = [];
+
+    if (window && window.__TENANT__) {
+        title.push(window.__TENANT__);
+    } else {
+        title.push('default');
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** Does it mean that on server side, the name of the tenant will always be `default` ?

Even when not on the `default` tenant.
Wouldn't it be better to set another name  (best: put the real tenant name).‏